lib/google_scraping_web/templates/keyword/show.html.heex

```diff
@@ -1,3 +1,73 @@
-<h2>Keywords details <%= @keyword.name %></h2>
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item">
+      <%= link(gettext("Keywords"),
+        to: Routes.keyword_path(@conn, :index)
+      ) %>
+    </li>
+    <li class="breadcrumb-item active" aria-current="page">
+      <%= gettext("Keyword '%{name}' details", %{name: @keyword.name}) %>
+    </li>
+  </ol>
+</nav>
 
-<p>Coming soon!</p>
+<div class="card">
+  <div class="card-header text-center">
+    <h2><%= @keyword.name %></h2>
+  </div>
+  <div class="card-body">
```

Comment:
Should we also have a feature test for this as well? 🤔

Reply:
We have this test https://github.com/Nihisil/nimble-certification-elixir/blob/feature/20-display-keywords-info/test/google_scraping_web/controllers/keyword_controller_test.exs#L90

Do we need to add something else?

Reply:
Is there a test to assert that the keyword information would be displayed on the page? Like URL count etc.

Reply:
Added it here: https://github.com/Nihisil/nimble-certification-elixir/pull/50/commits/a144d5471c8c36090a80588410e9e23fe5e2b102

With that we can be sure that stat section is dysplaying on the page.